vite.config.js

```diff
@@ -5,11 +5,12 @@ import utcPlugin from 'dayjs/plugin/utc.js';
 import { defineConfig } from 'vite';
 import browserslistToEsbuild from 'browserslist-to-esbuild';
 
-import babel from 'vite-plugin-babel';
 import eslint from 'vite-plugin-eslint';
 import { nodeResolve } from '@rollup/plugin-node-resolve';
 import stylelint from 'vite-plugin-stylelint';
 import { VitePWA } from 'vite-plugin-pwa';
+import istanbul from 'vite-plugin-istanbul';
+import { COVER_INCLUDE, COVER_EXCLUDE } from './config/coverage.js';
 import yaml from './config/vite-plugin-yaml.js';
 import handlebars from './config/vite-plugin-handlebars-loader.js';
 import inlineHbsCompile from './config/vite-plugin-inline-handlebars.js';
```

Comment:
This is unrelated to the changes in this PR.

But should we rename these handlebars plugins? To make it more clear that one is for compiling `.hbs` files & the other is for inline hbs? 

Like `hbsFileCompile` & `hbsInlineCompile`. Or something like that.